frontend/pages/DashboardPage/DashboardPage.tsx

```diff
@@ -896,7 +887,7 @@ const DashboardPage = ({ router, location }: IDashboardProps): JSX.Element => {
             name="platform-filter"
             value={selectedPlatform || ""}
             className={`${baseClass}__platform-filter`}
-            options={platformOptions}
+            options={[...PLATFORM_DROPDOWN_OPTIONS]}
```

Comment:
I don't think we need to spread, can just pass in directly:
`options={PLATFORM_DROPDOWN_OPTIONS}`